missions/templates/missions/mission_list.html

```diff
@@ -1,28 +1,39 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Missions Overview</title>
-    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-</head>
-<body>
+{% extends "base-missions.html" %}
+{% load bootstrap4 %}
+{% load fontawesome_5 %}
 
-    <div class="container">
-        <h1>Missions</h1>
-        <ul>
+{% block content %}
+<h1>Missions</h1>
+{% if object_list %}
+<table class="table table-hover">
+    <thead>
+        <tr>
+            <th>#</th>
+            <th>Name</th>
+            <th>Period</th>
+            <th>Ship</th>
+            <th></th>
```

Comment:
Hier würde ich auch noch "Actions" als Header einfügen, damit es analog zum Assignment im mission_detail.html ist. Alternativ könnte man auch in beiden Formularen "Actions" weg lassen.

Reply:
fixed with 7ea4c7d